File: recaudacion/2.Codigo/frontend/src/app/core/models/kardex.ts
export class Kardex {
    kardexId:number;
    documento: string;
    anioPecosa: number;
    numeroPecosa: number;
    fecha: any;
    entradaDocumento: string;
    entradaDel: number;
    entradaAl: number;
    entradaTotal: number;
    salidaDocumento: string;
    salidaDocumentoNumero: string;
    salidaDel: string;
    salidaAl: string;
    salidaTotal: number;
    saldo: number;
    index: number;
}

export class KardexSaldo {
    codigo: string;
    descripcion: string;
    kardexs: Kardex[];
}

File: recaudacion/2.Codigo/backend/RecaudacionApiReporte/Domain/Kardex.cs
using System;

namespace RecaudacionApiReporte.Domain
{
    public class Kardex
    {
        public int KardexId { get; set; }
        public string Documento { get; set; }
        public int AnioPecosa { get; set; }
        public int NumeroPecosa { get; set; }
        public DateTime Fecha { get; set; }
        public string EntradaDocumento { get; set; }
        public int EntradaDel { get; set; }
        public int EntradaAl { get; set; }
        public int EntradaTotal { get; set; }
        public string SalidaDocumento { get; set; }
        public string SalidaDocumentoNumero { get; set; }
        public int SalidaDel { get; set; }
        public int SalidaAl { get; set; }
        public int SalidaTotal { get; set; }
        public int Saldo { get; set; }
    }
}
